### PR TITLE
Fix class name in logging an unhandled exception

### PR DESCRIPTION
**Pull request by lgonczar**
### Change description ###

Fix class name in logging an unhandled exception

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sendletter/config/ThreadPoolConfig.java [hunk 1]
```diff
@@ -46,7 +46,7 @@ public TaskScheduler sendLetterTaskScheduler() {
         ThreadPoolTaskScheduler taskScheduler = new SendLetterTaskScheduler();
         taskScheduler.setThreadNamePrefix("SendLetterTask-");
         taskScheduler.setErrorHandler(t -> {
-            log.error("Unhandled exception during task. {}: {}", t.getClass(), t.getMessage(), t);
+            log.error("Unhandled exception during task. {}: {}", t.getClass().getName(), t.getMessage(), t);
```
Comment (doncem) [inline src/main/java/uk/gov/hmcts/reform/sendletter/config/ThreadPoolConfig.java]: `getName` or `getSimpleName`?

**Comment by doncem** [comment]
Re-opening as it looks like a good fix to me. Please close again if you disagree

**Comment by rkondratowicz** [comment]
can this be merged @lgonczar

**Comment by lgonczar** [comment]
I created this PR based on a false observation. Class has a proper `toString` implementation, so I came to the conclusion that this PR was unnecessary.

**Comment by doncem** [comment]
Yes, classes usually get serialised with package info as would `.getName()` I believe. Doesn't hurt to use actual method rather than trusting some serialiser 🙂. Happy with this method or `getSimpleName()`. Close PR with reason why so it's not reopened

**Pull request closed without merging**